stdlib/2and3/_typeshed.pyi

```diff
@@ -13,14 +13,14 @@
 # is not used, types from this module must be quoted.
 
 import sys
-from typing import AnyStr, Text, Union
+from typing import Text, Union
 
 # StrPath and AnyPath can be used in places where a
 # path can be used instead of a string, starting with Python 3.6.
 if sys.version_info >= (3, 6):
     from os import PathLike
     StrPath = Union[str, PathLike[str]]
-    AnyPath = Union[str, bytes, PathLike[AnyStr]]
+    AnyPath = Union[str, bytes, PathLike[str], PathLike[bytes]]
```

Comment:
The name `AnyPath` is mildly confusing because `AnyStr` is a typevar and this similarly named alias isn't. Not sure what to call it instead though.

Reply:
Well, `Any` is not a typevar either. We could go with just `Path`, but that is confusing with `pathlib.Path`. Maybe `EitherPath`, but I don't find that particularly clear. (I'm not suggesting `WhateverPath`, although it's kind of fitting. :) )

Reply:
I know at least the path modules (ntpath, posixpath, os.path, etc) also defines a BytesPath, is that worth moving here now, and updating those files?
For naming, I'm personally okay with `AnyPath`, as I can't come up with a clearer name

Reply:
I added `BytesPath` for consistency.